core/src/main/java/org/openstack4j/api/networking/NetworkingService.java

```diff
@@ -58,7 +59,12 @@
      * @return the LBaaS service
      */
     LoadBalancerService loadbalancers();
-    
+
```

Comment:
@vinodborole yes I haven't implemented yet. Wanted to get the OK to add lbaas v2 support before I started.